### PR TITLE
Fix CS on 4.4

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | n/a 
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Review by smithandre — APPROVED** [review]

File: src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php [hunk 1]
```diff
@@ -316,7 +316,7 @@ public function testLoadChoicesForValuesLoadsOnlyChoicesIfSingleIntId()
         $this->assertSame(
             [4 => $this->obj3, 7 => $this->obj2],
             $loader->loadChoicesForValues([4 => '3', 7 => '2']
-        ));
+            ));
```
Comment (derrabus) [inline src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php]: That does not look right. 👀

Comment (fabpot) [reply src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php]: At this point, I want to trust the fixer and not bother too much (even if I agree with you). So, not sure what we want here :(

Comment (fabpot) [reply src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php]: To add to my previous comment, I have rejected changes that look very wrong. This one is up for discussion, I suppose. Maybe reverting this is better indeed.

Comment (derrabus) [reply src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php]: This should be:

```php
        $this->assertSame(
            [4 => $this->obj3, 7 => $this->obj2],
            $loader->loadChoicesForValues([4 => '3', 7 => '2'])
        );
```

The closing bracket of the `loadChoicesForValues` call should be on the same line as the opening one. Not sure if it's worth fixing the fixer here though. We could fix that code manually and the fixer should be happy.

Comment (fabpot) [reply src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php]: Oh, you're right, fixed now.

**Review by stof — APPROVED** [review]

File: src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3HorizontalLayoutTest.php [hunk 2]
```diff
@@ -21,7 +21,7 @@ public function testLabelOnForm()
         $html = $this->renderLabel($view);
 
         $this->assertMatchesXpath($html,
```
Comment (HeahDude) [inline src/Symfony/Bridge/Twig/Tests/Extension/AbstractBootstrap3HorizontalLayoutTest.php:23]: To keep this change this should be full multilines args:
```suggestion
        $this->assertMatchesXpath(
            $html,
            '...
```
or 
```suggestion
        $this->assertMatchesXpath($html, '
...
```

File: src/Symfony/Component/Console/Tests/Formatter/OutputFormatterTest.php [hunk 3]
```diff
@@ -279,7 +279,7 @@ public function testContentWithLineBreaks()
 <info>
 some text</info>
 EOF
-        ));
+            ));
```
Comment (HeahDude) [inline src/Symfony/Component/Console/Tests/Formatter/OutputFormatterTest.php:282]:
```suggestion
            )
        );
```
?

**Pull request merged**